ReferenceTests/src/tests/figures_and_makielayout.jl

```diff
@@ -340,3 +340,15 @@ end
     translate!(vl, 1, 0, 0)
     f
 end
+
+@reference_test "PolarAxis fontsize from :Axis" begin
+    fig = Figure()
+    ax = PolarAxis(fig[1, 1], rticklabelsize = 35, thetaticklabelsize = 65)
+    fig
+end
```

Comment:
I don't think this is testing PolarAxis fontsize from Axis - to do that, you would do something like:
```suggestion
@reference_test "PolarAxis fontsize from :Axis" begin
    fig = Figure(; Axis = (; xticklabelsize = 35, yticklabelsize = 65))
    ax = PolarAxis(fig[1, 1])
    fig
end
```

and make sure it works that way!  There are already tests for the direct syntax.

Reply:
Would it be wise to add a reference test for setting the fontsize via theme aswell? And furthermore, would it be a good idea to move the test for direct syntax to the other tests so that they are grouped together?

Reply:
Actually I think this would be better as unit tests. There is already a refimg test for decorations that should confirm that changing r-/thetaticklabelsize has the desired effect. So here you just need to confirm that the PolarAxis is correctly inheriting values. Something like
```julia
fig = Figure(; Axis = (; xticklabelsize = 35, yticklabelsize = 65))
ax = PolarAxis(fig[1, 1])
@test ax.thetaticklabelsize[] == 65
@test ax.rticklabelsize[] == 35
```
should be enough for that.

Btw, should we switch this around to be x -> theta and y -> r? Originally we had (r, theta) as the default argument order and matched that to (x, y), but now it's (theta, r).

Reply:
I think we may as well switch it then, so things stay consistent.